ci/pipelines/pr.yml

```diff
@@ -46,48 +46,58 @@ jobs:
     plan:
       - aggregate:
         - do:
-          - get: govwifi-admin
+          - get: src
             resource: pull-request
             trigger: true
             version: every
           - <<: *update-status-pending
-        - get: mysql-image
+
+        # fetch each image and place in the docker-cache directory
+        - get: docker-cache/mysql-image
+          resource: mysql-image
           params: {format: oci}
-        - get: ruby-image
+        - get: docker-cache/ruby-image
+          resource: ruby-image
           params: {format: oci}
-        - get: nginx-image
+        - get: docker-cache/nginx-image
+          resource: nginx-image
```

Comment:
Do we need Nginx?  Pretty sure we just use Puma.

Reply:
it's used by the `fake-register`

Reply:
Ah yes ok